src/index.js

```diff
@@ -1,4 +1,6 @@
+import dotenv from 'dotenv';
 import app from './app';
 
-const port = process.env.PORT || 4000;
+dotenv.config();
+const port = process.env.PORT || 5000;
```

Comment:
Any reason for changing the port from 4000 to 5000?

Reply:
Since I have used 4000 in .env that line means that if 4000 is taken 5000 will be then used